test/virtual/include/openenclave/edger8r/common.h

```diff
@@ -146,11 +152,11 @@ OE_INLINE oe_result_t oe_add_size(size_t* total, size_t size)
 #define OE_COPY_AND_SET_IN_OUT_POINTER(argname, argcount, argsize, argtype)    \
     if (_pargs_in->argname)                                                    \
     {                                                                          \
-        size_t _size;                                                          \
-        OE_COMPUTE_SIZE(argcount, argsize, _size);                             \
+        size_t _size = 0;                                                      \
+        OE_COMPUTE_ARG_SIZE(_size, argcount, argsize);                         \
```

Comment:
Why not just consolidate the two calls for `OE_COMPUTE_ARG_SIZE` and `OE_ADD_SIZE` into `OE_ADD_ARG_SIZE` here?

Similarly lines 174, 186, 196, 222

Reply:
The `_size` is required by the memcpy. https://github.com/openenclave/oeedger8r-cpp/blob/b32b598d5fb2acb4bfa2b20c192d088d99069349/test/virtual/include/openenclave/edger8r/common.h#L165

Reply:
Right, missed that.

Also didn't realize that OE_ADD_SIZE is really OE_ADD_SIZE_ALIGNED and that might be useful to call out in the future.